test/e2e/BUILD.bazel

```diff
@@ -1,7 +1,24 @@
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
+load("@angular//:index.bzl", "protractor_web_test_suite")
 
 ts_library(
     name = "e2e",
     testonly = 1,
     srcs = glob(["*.ts"]),
 )
+
+protractor_web_test_suite(
```

Comment:
maybe we should add comments with links to API docs now that they are published

Reply:
where did those get published to?